corehq/sql_db/management/commands/print_max_id_of_pg_tables.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand
+from django.db import connections
+
+
+def get_sequences(cursor):
+    cursor.execute("SELECT c.relname FROM pg_class c WHERE c.relkind = 'S';")
+    return [r for (r, ) in cursor.fetchall()]
+
+
+def get_last_values(sequences, cursor):
+    # list of sequences
+    select_statements = [
+        '(SELECT last_value FROM {})'.format(s)
+        for s in sequences
+    ]
+    query = " UNION ALL ".join(select_statements)
+    cursor.execute(query)
+    result = [r for (r,) in cursor.fetchall()]
+    result = dict(zip(sequences, result))
+    return result
+
+
+class Command(BaseCommand):
+    help = """List last-value of all postgresql sequences. Useful to know
+           if any primary keys are near their max limit
+           """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'dbname',
+            help='Django db alias'
+        )
+        parser.add_argument(
+            '--minval',
+            type=int,
+            dest='minval',
+            help='print only sequences where the last_value is more than this provided minvalue'
+        )
```

Comment:
what would you put here in practice?

you should be able to get the sequence min / max value from postgres

Reply:
Ok, modified to pull max_value and display it in output.